wandb/proto/wandb_internal.proto

```diff
@@ -1361,3 +1363,36 @@ message JobInputRequest {
   repeated JobInputPath exclude_paths = 3;
   string input_schema = 4;
 }
+
+// ServerFeatures related protobufs
+
+enum ServerFeature {
+  LARGE_FILENAMES = 0;
+  ARTIFACT_TAGS = 1;
+  CLIENT_IDS = 2;
+}
+
+/**
+ * ServerFeatureRequest: request to the backend server for the features it supports.
+ * including the details of the feature provided by the server.
+ */
```

Comment:
nit: since this isn't Go, I would exclude the subject of the comment from the comment itself, since it's redundant

```
// A request to the backend to check supported features.
```